src/Services/ApiProvider/Client/Order/Order.php

```diff
@@ -17,6 +17,7 @@ public function getFilteredCollectionWithHttpInfo(OrderFilter $filter)
             $filter->getBillingAddressFullName(),
             $filter->getBillingAddressCompanyName(),
             $filter->getPaymentComment(),
+            $filter->getPaymentReference(),
```

Comment:
Djo : changed the position so it matches the order of arguments in order-sdk::OrderApi::getOrderCollection()